spec/services/character/relocater_spec.rb

```diff
@@ -0,0 +1,133 @@
+RSpec.describe Character::Relocater do
+  include ActiveJob::TestHelper
+
+  let(:relocator) { Character::Relocater.new(create(:admin_user)) }
```

Comment:
It looks like the relocator is meant to take an ID, not the user object - does this work right?

Reply:
That is weird! why are they passing!

Reply:
...apparently `Model.find_by(id: instance_of_that_model)` Works By Rails Magic